definitions/README.md

```diff
@@ -0,0 +1,92 @@
+# Dataset definitions
+
+Any models that have not been explicitly trained on the control tasks requires the proper instruction prompt to perform RL(Reinforcement Learning) tasks properly. To do this, we define the necessary information for each dataset in this directory.
+
+<br/>
+
+---
+
+### Category
+
+- **Descriptions**: These are descriptions of each dataset and environment. Any required information to understand a certain environment (e.g. the objects in the observation, the goal of the task, the restriction when performing a task, etc.) should be defined. (**Required**)

```

Comment:
please note that there are many datasets that didn't explicitly include any of these information like Berkeley Autolab UR5 did.
For example a dataset like RT-1, is a very generic dataset that focuses on a certain skills, and hundreds of different tasks with different environments within each task, and this is how i interpreted the descriptions
```py
    "rt_1_robot_action": {
        "pick and place items": [  # 130 tasks
            "Lift the object off the surface."
        ],
        "move object near another object": [  # 337 tasks
            "Move the first object near the second."
        ],
        "place objects up-right": [  # 7 tasks
            "Place an elongated object up-right."
        ],
        "open a drawer": [  # 3 tasks
            "Open any of the cabinet drawers."
        ],
        "close a drawer": [  # 3 tasks
            "Close any of the cabinet drawers."
        ],
        "place object into receptacle": [  # 84 tasks
            "Place an object into a receptacle."
        ],
        "pick object into receptacle and place on the counter": [  # 162 tasks
            "Pick an object up from a location and then place it on the counter."
        ]
    },
```
i tried to make the descriptions as generic as possible so they can be used for all of the environments, does that make sense for you? if yes, it's probably worth adding a note here about how the descriptions should be generic to work for different environments in case the dataset is too generic (has lots of tasks)